apollos-church-api/src/server.js

```diff
@@ -60,6 +60,11 @@ const apolloServer = new ApolloServer({
 
 const app = express();
 
+// health check
+app.get('/health', (req, res) => {
+  res.send('ok');
+});
+
```

Comment:
@vinnyjth what do you think about this? simple and elegant. and I constantly wanted something like this at newspring. main purpose is to validate the deploy.

Reply:
@vinnyjth you see this?

Reply:
Thanks for the ping, this is good!